scripts/check-image-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+tmpfile="$(mktemp)"
+trap 'rm -f "${tmpfile}"' EXIT
+
+git grep -E -o -h '(stackrox|scanner)-(build|test)-0.[0-9]+\.[0-9]' | grep -E -o '0.[0-9]+\.[0-9]' | sort -u >"$tmpfile"
+
+if [[ "$( wc -w < "$tmpfile" )" -eq 1 ]]
+ then
+	exit 0
+fi
+
+echo >&2 "Found multiple image versions:"
+cat >&2 "$tmpfile"
+
```

Comment:
Maybe something like:
```suggestion
echo >&2 "See $0 for the command which finds them."
```
?